src/account.ts

```diff
@@ -49,7 +130,7 @@ export const isZeroAddress = function(hexAddress: string): boolean {
  */
 export const toChecksumAddress = function(hexAddress: string, eip1191ChainId?: number): string {
```

Comment:
Can we eventually also add an [EIP-1191](https://eips.ethereum.org/EIPS/eip-1191) link here on the respective parameter comment for convenience?

Reply:
added in https://github.com/ethereumjs/ethereumjs-util/pull/275/commits/e48478d3449fed6142cd6f611dffab5d4ed46ee3